### ISSUE TITLE
Comms Issue # 13

### ISSUE THREAD

**Issue by coleweusthem** [asc-csa/AlouetteApp#39]
The element label must not appear as a descendant of the a element.

The element input must not appear as a descendant of the a element.

The element button must not appear as a descendant of the a element.

```
<a href="https://www.canada.ca/en/sr/srb.html">↩
<form action="#" method="post" name="cse-search-box" role="search" class="form-inline">↩
<div class="form-group">↩
<label for="wb-srch-q" class="wb-inv">Search Canada.ca</label>↩
<input id="wb-srch-q" list="wb-srch-q-ac" class="wb-srch-q form-control" name="q" type="search" value="" size="34" maxlength="170" placeholder="Search Canada.ca">↩
<datalist id="wb-srch-q-ac"> </datalist>↩
</div>↩
<div class="form-group submit">↩
<button type="submit" id="wb-srch-sub" class="btn btn-primary btn-small" name="wb-srch-sub"><span class="glyphicon-search glyphicon"></span><span class="wb-inv">Search</span></button>↩
</div>↩
</form>↩
</a>↩

```

**Comment by coleweusthem** [comment]
With the new english header this is fixed, but I need to implement a french header for it to fixed in french

**Comment by coleweusthem** [comment]
addressed in #98
